packages/frametricks.lua

```diff
@@ -174,7 +174,7 @@ SILE.registerCommand("float", function (options, content)
   local hbox = SILE.call("hbox", {}, content)
   table.remove(SILE.typesetter.state.nodes) -- steal it back
   local heightOfPageSoFar = SILE.pagebuilder:collateVboxes(SILE.typesetter.state.outputQueue).height
-  if SILE.length(heightOfPageSoFar + hbox.height - SILE.typesetter:getTargetLength()) > 0 then
+  if SILE.length.make(heightOfPageSoFar + hbox.height - SILE.typesetter:pageTarget()).length > 0 then
```

Comment:
This is a bad revert, you the measurements system was refactored _after_ this, you can't just start using the `.length` property again like this.